docs/sources/setup-grafana/configure-security/configure-authentication/github.md

```diff
@@ -101,6 +101,33 @@ allow_sign_up = true
 allowed_organizations = github google
 ```
 
+### Map roles
+
+You can use GitHub OAuth to map roles. During mapping, Grafana checks for the presence of a role using the [JMESPath](http://jmespath.org/examples.html) specified via the `role_attribute_path` configuration option.
+
+For the path lookup, Grafana uses JSON obtained from querying GitHub's API [`/api/user`](https://docs.github.com/en/rest/users/users#get-the-authenticated-user=) endpoint and a `groups` key containing all of the user's teams. The result of evaluating the `role_attribute_path` JMESPath expression must be a valid Grafana role, for example, `Viewer`, `Editor` or `Admin`. For more information about roles and permissions in Grafana, refer to [Roles and permissions]({{< relref "../../../administration/roles-and-permissions/" >}}).
+
+An example Query could look like the following:
+
+```bash
+role_attribute_path = [login==octocat] && 'Admin' || 'Viewer'
+```
+
+This allows the user with login "octocat" to be mapped to the `Admin` role,
+but all other users to be mapped to the `Viewer` role.
+
+#### Map roles using groups
+
+Groups can also be used to map roles. For instance,
+if you have a group called 'example-group' you can use the following snippet to
```

Comment:
nit: Could we use GitHub specific more real example, as I spent fare amount of time trying to understand why my setup is not working, until I have realized I am missing @ at the beginning. So perhaps @example-group ? Or maybe even describe with human words how are we building the shorthand version [here](https://github.com/grafana/grafana/pull/52407/files#diff-5bd5381c7a8841520acb8a969432c1454f7b5888e00d8f68ef1a62807cfde436R244).